templates/offcanvas.mustache

```diff
@@ -0,0 +1,22 @@
+{{#regions.offcanvas.hasblocks}}
```

Comment:
This file does not have any template heading (which includes a description of the template and an example context)